### PR TITLE
6323-RubSmalltalkEditor-meta-j-shortcut-not-working

### LINKED ISSUE #6323: RubSmalltalkEditor meta-j shortcut not working
**Describe the bug**

`RubSmalltalkEditor class >> buildShortcutsOn:` maps `jumpToNextKeywordOfIt: true` to `$j meta`. Pressing `Ctrl-j` doesn't work.

Note that pressing `Ctrl-Shift-j` (`jumpToNextKeywordOfItL false`) *does* work - it jumps to the previous keyword.

**Version information:**
 - OS: Kubuntu 19.10
 - Version: 8.0.0

I've fixed this, patch coming soon!

### PR DESCRIPTION
**Pull request by MarcusDenker**
fixes #6323 as described in the issue tracker entry

### CODE REVIEW

**Comment by MarcusDenker** [comment]
a green CI!

File: src/Rubric/RubSmalltalkEditor.class.st [hunk 1]
```diff
@@ -96,7 +96,7 @@ RubSmalltalkEditor class >> buildShortcutsOn: aBuilder [
 		do: [ :target | target editor jumpToNextKeywordOfIt: true ]
 		description: 'Jump to next keyword'.
 		
-	(aBuilder shortcut: #jumpToNextKeywordOfIt)
+	(aBuilder shortcut: #jumpToPrevKeywordOfIt)
 		category: RubSmalltalkEditor name
 		default: $j shift meta
 		do: [ :target | target editor jumpToNextKeywordOfIt: false ]
```
Comment (pavel-krivanek) [inline src/Rubric/RubSmalltalkEditor.class.st:102]: Here it is not consistent with the editor message.

Comment (MarcusDenker) [reply src/Rubric/RubSmalltalkEditor.class.st:102]: yes, the boolean changes it into a backjump. Bad method name... 

jumpToNextKeywordOfIt: isForwardJump "Jump to the next/previous keyword after the cursor"

Comment (pavel-krivanek) [reply src/Rubric/RubSmalltalkEditor.class.st:102]: ah :-)

**Review by pavel-krivanek — APPROVED** [review]

**Pull request merged**